packages/cli/src/databases/migrations/common/1727369628621-StandardizeTableNames.ts

```diff
@@ -0,0 +1,34 @@
+import type { MigrationContext } from '@/databases/types';
+
+export class StandardizeTableNames1727369628621 {
+	async up({ queryRunner, escape }: MigrationContext) {
+		const tableNameMap = {
+			annotation_tag_entity: 'annotation_tag',
+			credentials_entity: 'credential',
+			event_destinations: 'event_destination',
+			execution_annotation_tags: 'execution_annotation_tag',
+			execution_annotations: 'execution_annotation',
+			execution_entity: 'execution',
+			installed_nodes: 'installed_node',
+			installed_packages: 'installed_package',
+			settings: 'setting',
+			shared_credentials: 'shared_credential',
+			tag_entity: 'tag',
+			user_api_keys: 'user_api_key',
+			variables: 'variable',
+			webhook_entity: 'webhook',
+			workflow_entity: 'workflow',
+			workflows_tags: 'workflow_tag',
+		} as const;
+
+		await Promise.all(
+			// eslint-disable-next-line @typescript-eslint/promise-function-async
```

Comment:
You can remove this rule disable if you mark the function inside map as async

Reply:
Did that but then got a different error, which is an unsafe return type. Switching to runQuery seems to fix the issue,

![CleanShot 2024-10-01 at 16 38 24@2x](https://github.com/user-attachments/assets/62d467b4-dd36-4fd5-aa22-8b9a6e6e2376)

![CleanShot 2024-10-01 at 16 39 29@2x](https://github.com/user-attachments/assets/a386f98e-4de4-4380-b8bd-52ff2940f193)